src/v2/components/BlockLightbox/components/BlockLightboxMetadataPane/index.tsx

```diff
@@ -45,6 +48,14 @@ export default class BlockLightboxMetadataPane extends PureComponent {
     this.setState({ mode: 'resting', autoFocus: null })
   }
 
+  get canManage() {
+    if (this.props.block.__typename !== 'Channel') {
+      return this.props.block.can.manage
+    } else {
+      return false
+    }
+  }
+
```

Comment:
When converting to .ts, noticed this type-error -- `can.manage` doesn't exist on `Channel`, so this makes things a bit safer.

Reply:
Hm, this _should_ definitely be there on Channel always

Reply:
Yeah, it definitely should be there. I'll take a look..

Reply:
Cool, i'll revert this change then 👍

Reply:
https://github.com/aredotna/ervell/pull/2253/commits/ddbbac7c55e97418dcbbb1bbc05e469f48b9a1f7